src/app/grid/grid-export-visualization/grid-export-visualization.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="grid__wrapper">
+<igx-grid [data]="localData" [autoGenerate]="true" [showToolbar]="true" exportText="Export" width="100%" height="300px"
```

Comment:
The grid data currently looks half empty and repeating. It would look better if the columns are defined instead of using autoGenerate. The first columns should be the order related ones like OrderID, ProductName, Quantity, UnitPrice, etc.